src/views/catalog/customize/components/DiskSource/constants.ts

```diff
@@ -0,0 +1,21 @@
+import { V1beta1DataVolumeSource } from '@kubevirt-ui/kubevirt-api/kubevirt';
+
+export const PVC_DISK_SOURCE_NAME = 'pvc';
+export const HTTP_DISK_SOURCE_NAME = 'http';
+export const REGISTRY_DISK_SOURCE_NAME = 'registry';
+
+export type DISK_SOURCE_OPTIONS_IDS =
+  | typeof PVC_DISK_SOURCE_NAME
```

Comment:
no need for `typeof`

Reply:
without typeof i have this error `'DEFAULT_DISK_SOURCE' refers to a value, but is being used as a type here. Did you mean 'typeof DEFAULT_DISK_SOURCE'?ts(2749)` as DEFAULT_DISK_SOURCE and so on are const. Should I remove the constants and create an enum?